src/OpenTelemetry.Exporter.Geneva/Metrics/OtlpProtobuf/OtlpProtobufSerializer.cs

```diff
@@ -27,6 +27,10 @@ internal sealed class OtlpProtobufSerializer
 
     private readonly int prepopulatedHistogramDataPointAttributesLength;
 
+    private readonly byte[] prepopulatedExponentialHistogramDataPointAttributes;
```

Comment:
Could we use the same `byte[]` for all the data points: `NumberDataPoint`, `Histogram`, and `ExponentialHistogram`?

Reply:
No, because each attribute needs to prefixed with tag fieldNumber of attribute within the message. And unfortunately, that fieldNumber is not same for `NumberDataPoint`, `Histogram` and `ExponentialHistogram`.